bin/create_exercise.dart

```diff
@@ -6,87 +6,95 @@ import 'package:args/args.dart';
 import 'package:path/path.dart' show dirname;
 
 /// Constants
-const String scriptFilename = "create-exercise";
+const String scriptFilename = 'create-exercise';
 
 final parser = new ArgParser()
```

Comment:
Recommend: Remove all `new` keywords as they are optional in Dart 2.0+

Reply:
I was thinking about doing that in this PR, but I was afraid it would be cramming too much. But I can certainly do that.